### PR TITLE
Pan european/fix export and navigation

### LINKED ISSUE #366: panEuropean/master branch
- [ ] Create panEuropean/master branch
- In a new branch:
  - [ ] Add PanEuropean to the navigation menu and show table 1.1.a

### PR DESCRIPTION
**Pull request by sorja**
Resolves #366

### CODE REVIEW

File: webapp/app/dataExport/utils/panEuropeanMappings.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// TODO: replace with redux store navigation current prefix or similar
+export const tableMappings = {
+  forestArea: '1_1a',
+}
```
Comment (minotogna) [inline webapp/app/dataExport/utils/panEuropeanMappings.js:4]: why do we need this? can we use similar to FRA?

File: webapp/app/assessment/components/section/assessmentSectionView/assessmentSection.js [hunk 2]
```diff
@@ -24,7 +25,7 @@ const AssessmentSection = forwardRef((props, ref) => {
 
   const i18n = useI18n()
   const [printView, printOnlyTablesView] = usePrintView()
-  const disabled = useSelector(FraState.isSectionEditDisabled(sectionName))
+  const disabled = isPanEuropean(assessmentType) || useSelector(FraState.isSectionEditDisabled(sectionName))
```
Comment (minotogna) [inline webapp/app/assessment/components/section/assessmentSectionView/assessmentSection.js:28]: dont you get a hook warning in the console?

File: webapp/app/components/navigation/components/assessment/header/header.js [hunk 3]
```diff
@@ -12,14 +12,19 @@ import { useIsDataExportView } from '@webapp/components/hooks'
...
+  const {
+    assessment,
+    assessment: { type: assessmentType },
+    showSections,
+    setShowSections,
+  } = props
 
   const countryIso = useCountryIso()
-  if (useIsDataExportView()) {
+  if (useIsDataExportView() || assessmentType === 'panEuropean') {
```
Comment (minotogna) [inline webapp/app/components/navigation/components/assessment/header/header.js:27]: use assessment check, not the constant

File: common/assessment/assessment.js [hunk 4]
```diff
@@ -18,6 +19,7 @@ const getDeskStudy = R.propEq(keys.deskStudy, true)
 const getCanEditData = R.propEq(keys.canEditData, true)
 const getTablesAccess = R.propOr([], keys.tablesAccess)
 const isStatusChanging = R.pipe(getStatus, R.equals(assessmentStatus.changing))
+const isPanEuropean = R.equals(keys.panEuropean)
```
Comment (minotogna) [inline common/assessment/assessment.js:22]: rename this to isTypePanEuropean. you are not passing the assessment here, just a string

File: server/traditionalTable/api.js [hunk 5]
```diff
@@ -1,36 +1,37 @@
...
 
-  app.get('/traditionalTable/:countryIso/:tableSpecName', async (req, res) => {
+  app.get('/traditionalTable/:assessmentType/:countryIso/:tableSpecName', async (req, res) => {
     try {
-      const schemaName = await VersionService.getDatabaseSchema(req)
-      const result = await repository.read(req.params.countryIso, req.params.tableSpecName, schemaName)
+      const {
+        params: { assessmentType, countryIso, tableSpecName },
+      } = req
+      const schemaName = assessmentType === 'panEuropean' ? 'pan_european' : await VersionService.getDatabaseSchema(req)
```
Comment (minotogna) [inline server/traditionalTable/api.js:30]: use assessment check, not the constant

File: server/traditionalTable/mappings/panEuropean/index.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 const table11a = require('./table1_1a')
 
 module.exports = {
-  panEuropean11a: table11a,
+  // panEuropean11a: table11a,
+  table_1_1a: table11a,
```
Comment (sorja) [inline server/traditionalTable/mappings/panEuropean/index.js:5]: Could also be:

```
const table_1_1a = require('./table1_1a')
module.exports = {
  table_1_1a,
}
```

File: webapp/app/dataExport/utils/panEuropean.js [hunk 7]
```diff
@@ -0,0 +1,4 @@
+import { tableMappings } from '@webapp/app/dataExport/utils/panEuropeanMappings'
+
+export const getPanEuropeanTableMapping = (section) =>
```
Comment (sorja) [inline webapp/app/dataExport/utils/panEuropean.js:3]: This function is used to get the correct table mapping for server/traditionalTable/mappings/panEuropean/index.js

File: webapp/app/assessment/components/section/assessmentSectionView/assessmentSection.js [hunk 8]
```diff
@@ -36,7 +38,9 @@ const AssessmentSection = forwardRef((props, ref) => {
 
       <CustomHeader assessmentType={assessmentType} sectionName={sectionName} disabled={disabled} />
 
-      <Descriptions sectionName={sectionName} descriptions={descriptions} disabled={disabled} />
+      {!isTypePanEuropean(assessmentType) && (
```
Comment (minotogna) [inline webapp/app/assessment/components/section/assessmentSectionView/assessmentSection.js:41]: also GeneralComments should be disabled for pan European

File: server/traditionalTable/mappings/panEuropean/index.js (hunk 6 above)
Comment (minotogna) [reply server/traditionalTable/mappings/panEuropean/index.js:5]: yes that would be better

Comment (sorja) [reply server/traditionalTable/mappings/panEuropean/index.js:5]: But eslint will htrow an error for this - so we can keep it as is for now

File: webapp/app/assessment/components/section/assessmentSectionView/assessmentSection.js (hunk 8 above)
Comment (sorja) [reply webapp/app/assessment/components/section/assessmentSectionView/assessmentSection.js:41]: Should be handled automatically by:
`descriptions.comments &&`

**Pull request merged**